frontend/src/api/table.ts

```diff
@@ -9,25 +9,39 @@ import { setModelForFilterPredicateGroup } from "./slice";
 
 export async function getFilteredTable(
 	completeColumns,
+	filterModels: string[],
+	diffColumn: ZenoColumn,
 	filterPredicates: FilterPredicateGroup,
-	model: string,
 	sliceRange: [number, number],
-	sort: [ZenoColumn, boolean],
+	sort: [ZenoColumn | string, boolean],
 	tagIds: FilterIds,
 	filterIds?: FilterIds,
 	tagList?: Array<string>
 ) {
 	const requestedColumns = completeColumns.filter(
 		(c) =>
 			c.columnType !== ZenoColumnType.EMBEDDING &&
-			((c.columnType === ZenoColumnType.OUTPUT && c.model === model) ||
-				(c.columnType === ZenoColumnType.POSTDISTILL && c.model === model) ||
-				(c.columnType !== ZenoColumnType.OUTPUT &&
-					c.columnType !== ZenoColumnType.POSTDISTILL))
+			(filterModels.includes(c.model) || c.model === "")
 	);
+
+	// create diff columns for comparison view
```

Comment:
Do we always do this now? what if it's just the normal table? We still calculate this? probably should only be if in comparison view

Reply:
If it's not in comparison view, will pass undefined for diffColumn, so it won't be calculated I think.